internal/backup_operations/make_backup.go

```diff
@@ -203,3 +203,131 @@ func MakeBackup(
 
 	return backup, op, nil
 }
+
+func RetryBackup(
```

Comment:
I think I will try to merge `MakeBackup` and `RetryBackup` into one method. Now they are the same, except one accepts MakeBackupRequest, while other accepts TBWROperation.
I can essentially make a separate type `MakeBackupInternalRequest` just for the method, transfer data to it and make the call.

Then, it'll have an additional argument `reuseBackupId`, which will not spawn another Backup entity for retries of TBWR op.

Reply:
done